src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -85,17 +85,22 @@ class ProtoPropertyDecorator {
 
 
     boolean isMultiple() {
-        return hasValues()
+        return valuesCount > 1
+    }
+
+
+    ProtoValue getValue() {
+        innerProtoProperty.valuesList.first()
```

Comment:
Think we have two choices:
1. Maintain empty value
2. Truncate properties with no values

According to the JCR 2.0 spec at least, a property can have zero or more values which seems to be backed up by the node validation in CRX. https://docs.adobe.com/content/docs/en/spec/jcr/2.0/3_Repository_Model.html